src/client/driver/driver.js

```diff
@@ -339,18 +411,37 @@ export default class Driver {
         // but it should wait until the page is loaded before executing a command.
         this.readyPromise
             .then(() => {
+                if (this.dialogsMonitor && !isHandleDialogCommand(command))
```

Comment:
And please, add the `TODO:` comment why we check `if(this.dialogsMonitor)`. As I understand we are going to get rid of this when dialogs handling in iframes is implemented?